monitoring/prometheus/alertmanager.yml

```diff
@@ -0,0 +1,38 @@
+global:
+  resolve_timeout: 10s
+  slack_api_url: ''
+
+route:
+  receiver: 'slack-default-notifications'
+  group_interval: 10s
+  repeat_interval: 10s
+  routes:
+  - matchers: [group = api]
+    receiver: 'slack-apis-notifications'
+  - matchers: [group = instance]
+    receiver: 'slack-instance-notifications'
+
+receivers:
+  - name: 'slack-default-notifications'
+    slack_configs:
+      - channel: '#alert-manager-testing'
```

Comment:
@burnerlee where are you configuring credentials to access the `#alert-manager-testing` ?

Reply:
@Ram81 at the top of the file we have to mention the slack webhook url in slack_api_url key... I have not pushed the url in this PR as I thought it would be wrong to share the webhook on a public repo

Reply:
How are you planning to pass it for deployment?